### ISSUE TITLE
Make create new open modal/popup o.i.d instead of doing an Ajax call

### ISSUE THREAD

**Issue by gabn88** [yourlabs/django-autocomplete-light#704]
Is it possible to override the select2 so that it does not do an ajax call on the 'Create "asdasdf"', but instead does some custom javascript function on click (such as loading a modal or popup)?

I have been searching a lot, but could not find it so quickly. Thanks a lot!

**Comment by jpic** [comment]
Yeah that would be possible, by overriding the javascript and implementing the missing view. It's probably too much work for the value it would add to your project, have you considered using django-addanother instead ?

**Comment by gabn88** [comment]
With some tweaks it is working fine, except that the initialization script of django-autocomplete-light does not work with modals (it initialized select2 everytime (n) the modal is openened, which in turn leads to 2n ajax requests).

**Comment by gabn88** [comment]
If you want I can post a small tutorial on how to do this?

**Comment by jpic** [comment]
Sounds like a great idea !

**Comment by teconomix** [comment]
Can you please share your howot? I am looking for the same thing right now and have no idea on how to do it...

**Comment by teconomix** [comment]
Is it possible to use django-addantoher together with autocomplete-light or do I have to switch to django-select2?

**Comment by jpic** [comment]
You can do what you want, everything is possible ;)

That said, I think you should try django-select2, you need to try everything to know what's best !
